src/table/table.stories.ts

```diff
@@ -29,6 +31,49 @@ import { clone } from "../utils/utils";
 
 const en = require("./../../src/i18n/en.json");
 
+@Component({
+	selector: "app-table",
+	template: `
+		<ibm-table
+			[model]="model"
+			[size]="size"
+			[showSelectionColumn]="showSelectionColumn"
+			[striped]="striped"
+			(sort)="simpleSort($event)">
+		</ibm-table>
+	`
+})
+class TableStory implements OnInit, OnChanges {
+	@Input() model = new TableModel();
+	@Input() size = "md";
+	@Input() showSelectionColumn = true;
+	@Input() striped = true;
+	@Input() sortable = true;
+	@Input() simpleSort;
+
+	ngOnInit() {
+
```

Comment:
Unnecessary empty line.

Reply:
Updated.
I like the rename to `metadata`. I didn't like the name `additionalData` but couldn't come up with a better name.